docs/babel.rst

```diff
@@ -65,3 +65,40 @@ These are all the files you need in order to use `pyhf xml2json <cli.html#pyhf-x
 which will read all of the XML files and load the histogram data from the histogram cache.
 
 The ``HistFactory`` workspace in ``results/`` contains all of the information necessary to rebuild the XML files again. For debugging purposes, the ``pyhf`` developers will often ask for your workspace file, which means ``results/3b_tag21.2.27-1_RW_ExpSyst_36100_multibin_bkg/Excl_combined_DefaultMeasurement.root``. If you want to generate the XML, you can open this file in ``ROOT`` and run ``DefaultMeasurement->PrintXML()`` which puts all of the XML files into the current directory you are in.
+
+
+TRExFitter
+----------
+
+.. note::
+
+    For more details on this section, please refer to the ATLAS-internal `TRExFitter documentation <https://trexfitter-docs.web.cern.ch/trexfitter-docs/advanced_topics/pyhf/>`_.
+
+In order to go from ``TRExFitter`` to ``pyhf``, the good news is that the RooWorkspace files (``XML`` and ``ROOT``) are already made for you. For a given configuration which looks like
+
+.. code:: yaml
+
+    Job: "pyhf_example"
+    Label: "..."
+
+You can expect some files to be made like so:
+
+- ``pyhf_example/RooStats/pyhf_example.xml``
+- ``pyhf_example/RooStats/pyhf_example_Signal_region.xml``
+- ``pyhf_example/Histograms/pyhf_example_Signal_region_histos.root``
+
+These are all the files you need in order to use `pyhf xml2json <cli.html#pyhf-xml2json>`_. At this point, you could run
```

Comment:
It would be nice if the `pyhf xml2json` could be code stylized, but I don't think that and the hyperlink are possible at the same time(?) and the hyperlink is the important part.

Reply:
it's code style afterwards.